DoSomething/quasar
# Dag graph update

#### What's this PR do?
* Updates DBT code to accurately reflect model dependencies.
* Updates SQL code references for `campaign_info` model that were referencing non-existent vars.
* Few SQL formatting cleanups.
#### Where should the reviewer start?
https://github.com/DoSomething/quasar/pull/844/commits/93ae40f1a1fbb54a2eb214742d869da09960e067
#### How should this be manually tested?
Checkout local branch and look at updates dependency graph. I'd check each commit separately since I structured model changes together for ease of review.
#### Any background context you want to provide?
We had some out of date dependencies and SQL references that should now be up to date, taking current non-DBT queries as a reference point.
#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/169272883
